### PR TITLE
Site Pickup Change

### PR DESCRIPTION
**Pull request by Legend8-8**
Disabled Pickup on arty tubes to prevent bug where there are loaded into vehicles and made invisible.
Tested and confirmed cannot be picked up but can still be spiked.
@dijksterhuis

### CODE REVIEW

**Comment by dijksterhuis** [comment]
@Legend8-8 needs to be for all sites

**Comment by dijksterhuis** [comment]
ZPUs will be affected by this, and possibly static weapons too.

**Comment by Legend8-8** [comment]
Was going to question this but figured the arty was the only asset that would be taken from sites > @Legend8-8 needs to be for all sites

**Comment by Legend8-8** [comment]
@dijksterhuis Have added AA sites, following this 

- [x] Player gets killed within the site radius -- object is still hidden and attached to player's body

Only changed sites where the  asset itself is relevant to site completion

File: mission/functions/systems/sites/create/site/fn_sites_create_site_aa.sqf [hunk 1]
```diff
@@ -31,8 +31,12 @@ params ["_pos"];
 		private _sitePos = getPos _siteStore;
 		private _spawnPos = _sitePos;
 
-		private _objects = [_spawnPos] call vn_mf_fnc_sites_create_compositions_aa;
-
+		private _objects = [_spawnPos] call vn_mf_fnc_sites_create_compositions_aa;             
+       
+       {
```
Comment (cerebral-314) [inline mission/functions/systems/sites/create/site/fn_sites_create_site_aa.sqf]:
```suggestion
		{     
			_x setVariable ["vn_log_enablePickup", false];
		} forEach _objects;
```

Comment (cerebral-314) [reply mission/functions/systems/sites/create/site/fn_sites_create_site_aa.sqf]: Please fix your whitespacing
Tab generally does it for you... if you need help with setting up your coding environment please feel free to DM me

File: mission/functions/systems/sites/create/site/fn_sites_create_site_artillery.sqf [hunk 2]
```diff
@@ -31,6 +31,8 @@ params ["_pos"];
 
 		private _artyObjs = [_spawnPos] call vn_mf_fnc_sites_create_compositions_mortar;
 		{
+		    _x setVariable ["vn_log_enablePickup", false];
```
Comment (cerebral-314) [inline mission/functions/systems/sites/create/site/fn_sites_create_site_artillery.sqf]:
```suggestion
		_x setVariable ["vn_log_enablePickup", false];
```

Comment (cerebral-314) [reply mission/functions/systems/sites/create/site/fn_sites_create_site_artillery.sqf]: Here as well please

**Review by cerebral-314 — CHANGES_REQUESTED** [review]

**Review by cerebral-314 — APPROVED** [review]

**Pull request merged**